nautobot_golden_config/__init__.py

```diff
@@ -15,7 +15,7 @@ class GoldenConfig(PluginConfig):
     """Plugin configuration for the nautobot_golden_config plugin."""
 
     name = "nautobot_golden_config"
-    verbose_name = "Golden Configuration"
+    verbose_name = "Nautobot Golden Configuration"
```

Comment:
Want to say this will cause new job entries to be added in Nautobot 1.3 as i believe `verbose_name` is used as the `grouping` name when the job model is populated.  I think you would lose the history of the job because those db entries will be a different reference.  @glennmatthews is this accurate or am I misunderstanding?

Reply:
To the best of my recollection we don't use the plugin `verbose_name` in populating jobs. The `grouping` will default to the `name` module attribute set in `nautobot_golden_config/jobs.py`

Reply:
Thanks @glennmatthews wanted to validate.